app/controllers/sites/click_drilldowns_controller.rb

```diff
@@ -33,7 +33,7 @@ def document_mapping(doc)
     record << doc['request']
     record << doc['referrer']
     record << doc['vertical']
-    record << doc['modules']
+    record << Array(doc['modules']).first
```

Comment:
After a minute with irb, I see what you're doing here. Could you move it out to a function, so the function name makes that obvious? 'force_click_module_to_string' or something, perhaps.